tests/register.spec.coffee

```diff
@@ -1,9 +1,6 @@
-requestMock = require 'requestmock'
-mockery = require 'mockery'
-
-requestMock.log(false)
-mockery.enable(warnOnUnregistered: false)
-mockery.registerMock('request', requestMock)
+Promise = require 'bluebird'
+global.Promise = Promise
```

Comment:
Strictly speaking, we can drop this indeed (in other modules too, then). I think in the first place it was because of the older node versions.
The reason it's global is because `fetch` relies on the global Promise to be present.